README.md

```diff
@@ -90,7 +90,20 @@ $ make test
 
 ### Soure Code Layout
 
-Currently most of our source files are stored in our root directory. The provided `nginx-configparser` source files are in the nginx-configparser submodule which also contains the GTest submodule.
+The current layout of our source files is laid out below. The assets to be served by the static handler are stored in `assets/`. The server can be conifgured using the configs stored in our root directory. The provided `nginx-configparser` source files are in the nginx-configparser submodule which also contains the GTest submodule. `Deploy` is used during deployment into images by Docker.
```

Comment:
A few minor points: 

* Typo: "conifgured" -> "configured"
* Can we keep the casing of `deploy/` consistent to prevent confusion?
* You mentioned `nginx-configparser` and `gtest`, but not `cpp-markdown`. How about just noting that the "Other libraries" are in this repo as submodules?